app/javascript/components/admin/AdminTable.js

```diff
@@ -100,6 +86,54 @@ class AdminTable extends React.Component {
     }
   }
 
+  /**
+   * Handles the expand collapse of the buttons in the notes column.
+   * @param {Object} rowData - Data about the cell this filter is called on.
+   */
+  handleExpandClick = rowId => {
+    let ids = this.state.expandedNotesIds;
```

Comment:
`const` instead of `let`

Reply:
Resolved in https://github.com/SaraAlert/SaraAlert/pull/805/commits/6692b2c0557b7786044f99801d4bce25b73e6047